packages/webpack-cli/lib/utils/arg-parser.js

```diff
@@ -63,6 +64,11 @@ function argParser(options, args, argsOnly = false, name = '', helpFunction = un
             if (!option.multiple) {
                 // Prevent default behavior for standalone options
                 parserInstance.option(flagsWithType, option.description, option.defaultValue).action(() => {});
+                if (option.group === 'core' && option.type === Boolean) {
+                    // commander requires explicitly adding the negated version of boolean flags
+                    const negatedFlag = `--no-${option.name}`;
+                    parserInstance.option(negatedFlag, `negates ${option.name}`);
+                }
```

Comment:
I think we can improve it like:
```js
{
            name: 'hot',
            usage: '--hot',
            alias: 'h',
            type: Boolean,
            negative: true
            group: ADVANCED_GROUP,
            description: 'Enables Hot Module Replacement',
            link: 'https://webpack.js.org/concepts/hot-module-replacement/',
},      
```

Allow core flags have `negative: true`, I think default webpack-cli flags also should have `negative: true`

Reply:
Yes, It's better.

Reply:
should I make a PR to core or it's okay to handle this CLI side.?

Reply:
I think it's okay to handle on CLI side, since we're already adding groups and other things, should be fine.

Reply:
then we can move this change to a seperate PR.

Reply:
yes, ideally, are you on it? If not, I have bandwidth to pick up

Reply:
@anshumanv WIP

Reply:
Great 👍

Reply:
done :raised_hands: